flume/README.md

```diff
@@ -275,6 +275,8 @@ AGENT_NAME=cygnusagent
 LOGFILE_NAME=cygnus.log
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/fe88716eb5cebfe043b0dd91e5c0cc8f04146cd0